### PR TITLE
refactor: use fstrings instead of `str.format`

### PR DESCRIPTION
**Pull request by c0rydoras**

### CODE REVIEW

File: timed/employment/models.py [hunk 1]
```diff
@@ -228,9 +228,7 @@ class Meta:
 
     def __str__(self) -> str:
         """Represent the model as a string."""
-        return "{} ({} - {})".format(
-            self.user.username,
-            self.start_date.strftime("%d.%m.%Y"),
+        return f"{self.user.username} ({self.start_date:%d.%m.%Y} - {{}})".format(
```
[the remark this answers is not in the document]
Comment (c0rydoras) [reply timed/employment/models.py:231]: i'm not sure

[the remark this answers is not in the document]
Comment (c0rydoras) [reply timed/employment/models.py:231]: i'm not sure, i just didn't want to break it

[the remark this answers is not in the document]
Comment (c0rydoras) [reply timed/employment/models.py]: i just replaced it now with a `(self.end_date or date.today()):%d.%m.%Y`

[the remark this answers is not in the document]
Comment (hairmare) [reply timed/employment/models.py]: sounds fair, let's hope noone relies on a models `__str__` for more than debugging

**Review by hairmare — APPROVED** [review]

**Pull request merged**